lib/common_fw/common_fw.cpp

```diff
@@ -36,36 +41,34 @@ bool write_to_SD(std::string message, const char * file_name) {
  * Constructs packet in the following format:
  * {<sensor_ID>,<data1>,<data2>, ...,<dataN>|checksum}
  */
-String make_packet(int id, bool error) {
-  String packet_content = (String)id;
-  packet_content += ",";
+uint8_t make_packet(uint8_t id, bool error) {
```

Comment:
What is `error` for?

Reply:
error packets. Currently just used for sd init/write errors.

Reply:
I see. so if error is true, it sends the id and a zero byte?